api/tools.ts

```
@@ -461,47 +440,29 @@ export function registerTools(server: McpServer, apiKey: string): McpServer {
-      platform,
-      rawCalendarId,
-    }: {
-      uuid: string;
-      oauthClientId?: string;
-      oauthClientSecret?: string;
-      oauthRefreshToken?: string;
-      platform?: "Google" | "Microsoft";
-      rawCalendarId?: string;
-    }) => {
+    async ({ calendarId, name, type, config }) => {
+      let updateCalendarParams: UpdateCalendarParams = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'updateCalendarParams' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)